### ISSUE TITLE
Training problem

### ISSUE THREAD

**Issue by a-chance888** [haotian-liu/yolact_edge#95]
Why some files in the project can be imported torch, but some files cannot be imported torch?

**Comment by a-chance888** [comment]
eval.py can import torch,but train.py can't.

**Comment by haotian-liu** [comment]
Please submit a issue [here](https://github.com/haotian-liu/yolact_edge/issues/new?assignees=&labels=&template=error-report.md&title=) with clearer description and necessarily details, thanks.
